base_maintenance/views/maintenance_request_views.xml

```diff
@@ -12,6 +12,13 @@
                     <!--Empty, to be inherited by other modules.-->
                 </div>
             </xpath>
+            <field name="description" position="replace">
```

Comment:
With a `replace` in a view, a lower priority for the view is recommended

Reply:
Ok, already changed in a separate commit